test/Nancy.Tests/Unit/Diagnostics/CustomInteractiveDiagnosticsFixture.cs

```diff
@@ -26,7 +28,10 @@ public class CustomInteractiveDiagnosticsHookFixture
         public CustomInteractiveDiagnosticsHookFixture()
         {
             this.cryptoConfig = CryptographyConfiguration.Default;
-            this.objectSerializer = new DefaultObjectSerializer();
+            var ass = Assembly.GetExecutingAssembly();

```

Comment:
I don't think this'll work well on core.Let's use `typeof(CustomInteractiveDiagnosticsHookFixture).GetTypeInfo().Assembly` or something.